helm-base-versions/helm-chart-base-versions.sh

```diff
@@ -30,7 +30,7 @@ for NAMESPACE_ROW in $(echo "${NAMESPACES}" | jq -r '.items[] | @base64' ); do
     NOTIFICATION_ARRAY=()
 
     if [[ $MODE == "notify" ]]; then
-      python3 send-notification-to-slack.py "$SLACK_WEBHOOK" "$NAMESPACE" "$TEAM_SLACK_CHANNEL"
+      python3 send-notification-to-slack.py -slack_webhook "$SLACK_WEBHOOK" -namespace "$NAMESPACE" -slack_channel "$TEAM_SLACK_CHANNEL"
```

Comment:
these should be "--" I think, looking at the parser spec below

Reply:
^^ Yep need to be `--` inputs same as the one you have below this

Reply:
oh yeh true